index.html

```diff
@@ -526,14 +536,150 @@ <h3 id="installation-sec">
         DOM events <a>fired</a> by this specification use the <dfn>application
         life-cycle task source</dfn>.
       </p>
+      <section data-dfn-for="BeforeInstallPromptEvent">
+        <h3>
+          <code>BeforeInstallPromptEvent</code> Interface
+        </h3>
+        <pre class="idl">
+          [Constructor]
+          interface BeforeInstallPromptEvent : Event {
+              Promise&lt;PromptResponseObject&gt; prompt();
+          };
+          dictionary PromptResponseObject {
+            AppBannerPromptOutcome userChoice;
+          };
+        </pre>
+        <p>
+          The <dfn>BeforeInstallPromptEvent</dfn> is dispatched prior to
+          activating an <a>automated install prompt</a>, allowing a developer
+          to prevent the default action for an install prompt.
+        </p>
+        <div class="note">
+          The default action of the <a>BeforeInstallPromptEvent</a> is to
+          <a data-lt="presents an install prompt">present an automated install
+          prompt</a> to the end-user. Canceling the default action (via
+          <a
+          data-cite="DOM#dom-event-preventdefault"><code>preventDefault</code></a>)
+          prevents the user agent from <a data-lt="presents an install
+          prompt">presenting an automated install prompt</a> until a later time
```

Comment:
Ah I think you misinterpreted this sentence. (Marcos wrote it and I just got confused myself now.) It's not trying to prevent prompting later. It's trying to prevent prompting *now*, but saying that the UA can prompt again later. I've rewritten it for clarity. WDYT?